source/lac/petsc_block_sparse_matrix.cc

```diff
@@ -1,104 +0,0 @@
-// ---------------------------------------------------------------------
```

Comment:
Formally, no, but it is impossible to use this class without also using one of the deprecated vector classes so I went ahead and removed it too.